cpp/tests/io/parquet_test.cpp

```diff
@@ -3020,8 +3018,34 @@ TEST_F(ParquetReaderTest, DecimalRead)
       std::begin(col1_data), std::end(col1_data), validity_c1, numeric::scale_type{-11});
     cudf::test::expect_columns_equal(result.tbl->view().column(1), col1);
 
-    read_opts.set_columns({"dec20p1"});
-    EXPECT_NO_THROW(cudf_io::read_parquet(read_opts));
+    auto validity_c2 =
+      cudf::detail::make_counting_transform_iterator(0, [](auto i) { return i != 6 and i != 14; });
```

Comment:
Apologies for missing this on the first pass. 
We might consider rephrasing this:
```c++
auto validity_c2 = cudf::test::iterators::nulls_at({6, 14});
```